config.js

```diff
@@ -1218,6 +1226,7 @@ function _validate_executable(file_name) {
 module.exports.load_nsfs_nc_config = load_nsfs_nc_config;
 module.exports.reload_nsfs_nc_config = reload_nsfs_nc_config;
 
+set_nc_nsfs_config_to_env(config);
```

Comment:
if it's called from load_nsfs_nc_config(), why doing it here?

Reply:
the was a misunderstanding from my part. I thought this function was called whenever we reload the config. and I didn't want to go over the entire config object each time. but now I see there is the reload_nsfs_nc_config for that